src/main/kotlin/com/devironment/devironmentapi/wastebag/WasteBagService.kt

```diff
@@ -25,7 +25,7 @@ constructor(
                     wasteCategory = wasteBagRequest.wasteCategory,
                     dateOfDisposal = LocalDate.parse(wasteBagRequest.dateOfDisposal),
                     user = user,
-                    delivered = false
+                    delivered = wasteBagRequest.delivered
```

Comment:
I did set this to allways default false as the WasteBag should never be directly delivered by client calls.